float32[] j1_pos
float32[] j2_pos
float32[] j3_pos
float32[] j4_pos
float32[] j5_pos
float32[] j6_pos
float32[] j1_vel
float32[] j2_vel
float32[] j3_vel
float32[] j4_vel
float32[] j5_vel
float32[] j6_vel
float32[] time_from_start

